src/app/page.js

```diff
@@ -0,0 +1,295 @@
+'use client'
+import React, { useState } from 'react';
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import Toolbar from '@mui/material/Toolbar';
+import Container from '@mui/material/Container';
+import Button from '@mui/material/Button';
+import { Typography } from '@mui/material';
+import Badge from '@mui/material/Badge';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+
+
+const user = {
+    coins: 10000000,
+    diamonds: 20000
+};
+
+function updateDisplay() {
+  
+  let screenWidth = window.innerWidth;
+  let coinsElement = document.getElementById("userCoins");
+  let diamondsElement = document.getElementById("userDiamonds");
+  if(600 <= screenWidth && screenWidth < 800){
+    document.getElementById("logo").setAttribute("src", "https://uploads-ssl.webflow.com/61a8d1d6f4ba145b578385f7/62ade6202da6393f23edcca0_32x232.png");
+    document.getElementById("logo").setAttribute("width", "30px");
+    document.getElementById("smallScreen").style.display = "flex";
+    document.getElementById("bigScreen").style.display = "none";
+  }
+  else{
+  if (screenWidth < 600) {
+    document.getElementById("smallScreen").style.display = "flex";
+    document.getElementById("bigScreen").style.display = "none";
+
+
+    if(String(user.coins).length <= 6){
+    coinsElement.innerHTML =" &#128178; " + String(Math.floor(Number(user.coins) / 1000)) + "K";
+    }
+    else{
+    coinsElement.innerHTML =" &#128178; " + String(Math.floor(Number(user.coins) / 1000000)) + "M";
+    } // 1000000
+    if(String(user.diamonds).length <= 6){
```

Comment:
There are packages that handle numeric values for this, some I haven't used but check them out : 

numeral.js - https://numeraljs.com/
d3-format -  https://github.com/d3/d3-format